cmd/login.go

```diff
@@ -57,13 +57,22 @@ func init() {
 	lgCmd.PersistentFlags().StringVar(&loginCmdArgs.aadEndpoint, "aad-endpoint", "", "the Azure active directory endpoint to use for OAuth user interactive login")
 	// Use identity which aligns to Azure powershell and CLI.
 	lgCmd.PersistentFlags().BoolVar(&loginCmdArgs.identity, "identity", false, "log in using virtual machine's identity, also known as managed service identity (MSI)")
+	// Use SPN certificate to log in.
+	lgCmd.PersistentFlags().BoolVar(&loginCmdArgs.certificate, "certificate", false, "log in using a certificate as a form of Service Principal Name (SPN) auth")
+	lgCmd.PersistentFlags().BoolVar(&loginCmdArgs.secret, "secret", false, "log in using a client secret as a form of Service Principal Name (SPN) auth")
```

Comment:
Could this be --service-principal as Azure CLI [https://docs.microsoft.com/en-us/cli/azure/authenticate-azure-cli?view=azure-cli-latest](https://docs.microsoft.com/en-us/cli/azure/authenticate-azure-cli?view=azure-cli-latest);?